docs/orchestration.md

```diff
@@ -0,0 +1,3 @@
+# Orchestration
```

Comment:
I would delete this now, see above comment.

Reply:
Sorry, which part? Delete it yourself and I'll accept the change.